### PR TITLE
Merged develop into apartapp-develop

### PR DESCRIPTION
**Pull request by NerdyGriffin**

### CODE REVIEW

File: src/setupProxy.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
-const proxy = require('http-proxy-middleware');
+const {createProxyMiddleware} = require('http-proxy-middleware');
```
Comment (joshuaRogers24) [inline src/setupProxy.js:1]: I know this change may cause us trouble if we do not update the http-proxy-middleware dependency. It was a breaking change.

Comment (joshuaRogers24) [reply src/setupProxy.js:1]: We need to let the whole team know that, once we merge these changes in, they will definitely have to delete node_modules and reinstall

**Review by joshuaRogers24 — CHANGES_REQUESTED** [review]
Nice. We have some changes to make due to some updates. Can you make sure our Snackbar use matches how it works now?

File: src/components/Snackbar/index.js [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
+  CheckCircleOutlineOutlined,
+  ErrorOutline,
+  InfoOutlined,
+  ReportProblemOutlined,
+} from '@material-ui/icons/';
 import MuiAlert from '@material-ui/lab/Alert';
 
-const Alert = (props) => {
-  return <MuiAlert elevation={6} variant="filled" {...props} />;
+const defaultAlertIconMapping = {
```
[the remark this answers is not in the document]
Comment (NerdyGriffin) [reply src/components/Snackbar/index.js:11]: What do you mean "the old version"? 
It is a site-wide component, I'm not sure what you are actually trying to ask.
If you are worried about compatibility, it looks like the outward "interface" of the component is functionally identical to how it was before, it is just improvements to the implementation, probably more consistent styling implementation, and taking better advantage of some React-related features

File: src/setupProxy.js (hunk 1 above)
Comment (NerdyGriffin) [reply src/setupProxy.js:1]: Note by the title that this is a merge of stuff from develop into our branch. So these were changes already reviewed and approved before they got merged into develop, and not written by me

Comment (NerdyGriffin) [reply src/setupProxy.js:1]: Ah, I now understand what you mean in the second comment, that is definitely a good point, that we would expect the existing 'npm install' of our local clones of the repo to break once this branch gets merged into apartapp-develop.

File: src/components/Snackbar/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (joshuaRogers24) [reply src/components/Snackbar/index.js:11]: Okay, that is what I was wondering, thank you

File: src/setupProxy.js (hunk 1 above)
Comment (joshuaRogers24) [reply src/setupProxy.js:1]: right, thanks for trying to understand!

**Review by joshuaRogers24 — APPROVED** [review]

**Review by joshuaRogers24 — APPROVED** [review]
Nice.

**Pull request merged**